test/internalTest.js

```diff
@@ -343,6 +343,84 @@ mineflayer.supportedVersions.forEach((supportedVersion, i) => {
       })
     })
 
+    it('bed', (done) => {
+      const bedId = 26
+      const bed1Pos = vec3(0, 0, 1) // Foot part
+      const bed2Pos = vec3(0, 0, 4)
+      const bed3Pos = vec3(3, 0, 0)
+      const zombiePos = vec3(10, 0, 0)
+
+      bot.once('chunkColumnLoad', (columnPoint) => {
+        assert.doesNotThrow(() => { // Should sleep
+          bot.sleep(bot.blockAt(bed1Pos))
+        })
+
+        assert.throws(() => { // The bed is too far
+          bot.sleep(bot.blockAt(bed2Pos))
+        })
+
+        assert.throws(() => { // There are monsters nearby
+          bot.sleep(bot.blockAt(bed3Pos))
+        })
+
+        done()
+      })
+
+      server.once('login', (client) => {
+        bot.time.day = 18000
+        client.write('login', {
+          entityId: 0,
+          levelType: 'fogetaboutit',
```

Comment:
foget?

Reply:
I did just copy the code from the 'blockAt' test. Didn't notice!